archipelago/docs/en_Donkey Kong 64.md

```diff
@@ -10,22 +10,62 @@ Items which the player would normally acquire throughout the game have been move
 always able to be completed, but because of the item shuffle the player may need to access certain areas before they
 would in the vanilla game.
 
+In addition, the level entrances from DK Isles have been swapped around. This means that you may find Creepy Castle where the entrance for Jungle Japes normally is, and so on.
+
+Due to some current technical constraints brought on by the Archipelago framework, a number of changes have been made from the regular Donkey Kong 64 randomizer logic, including:
+- Tag Anywhere is considered in logic
+- Purchases from shopkeepers (Cranky/Funky/Candy) are free
```

Comment:
These last 3 aren't outside of regular DK64 Randomizer logic, so I'd advise rephrasing this to suggest that it's not a new thing for AP, but rather a settings limitation that you can't set shops to non-free costs (as with B. Lockers being free and medals set to 40)